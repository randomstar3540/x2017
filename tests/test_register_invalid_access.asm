FUNC LABEL 0
    MOV REG 5 VAL 8
    RET
