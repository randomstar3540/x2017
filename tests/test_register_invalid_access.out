ERROR: Invalid access on register!
